data/saas/dataset/hubspot_dataset.yml

```diff
@@ -504,49 +147,26 @@ dataset:
                 data_categories: [system.operations]
                 fidesops_meta:
                   data_type: string
-#      - name: users
-#        fields:
-#          - name: id
-#            data_categories: [user.unique_id]
-#            fidesops_meta:
-#              primary_key: True
-#              data_type: string
-#          - name: email
-#            data_categories: [user.contact.email]
-#            fidesops_meta:
-#              data_type: string
-#          - name: roleId
-#            data_categories: [system.operations]
-#            fidesops_meta:
-#              data_type: string
-#          - name: primaryTeamId
-#            data_categories: [system.operations]
-#            fidesops_meta:
-#              data_type: string
-#          - name: secondaryTeamIds
-#            data_categories: [system.operations]
-#            fidesops_meta:
-#              data_type: string[]
-#      - name: user_provisioning
-#        fields:
-#          - name: id
-#            data_categories: [user.unique_id]
-#            fidesops_meta:
-#              primary_key: True
-#              data_type: string
-#          - name: email
-#            data_categories: [user.contact.email]
-#            fidesops_meta:
-#              data_type: string
-#          - name: roleId
-#            data_categories: [system.operations]
-#            fidesops_meta:
-#              data_type: string
-#          - name: primaryTeamId
-#            data_categories: [system.operations]
-#            fidesops_meta:
-#              data_type: string
-#          - name: secondaryTeamIds
-#            data_categories: [system.operations]
-#            fidesops_meta:
-#              data_type: string[]
+     - name: users
```

Comment:
This user collection needs to be tabbed.

Reply:
thanks, my b